project1.css

```diff
@@ -1,30 +1,33 @@
 .navbar {
-	background-color: #7a7676;
+	background-color: #fc7b03;
 	display: flex;
 	justify-content: center;
+	width: 100%;
```

Comment:
i don't think this line of code is doing anything - i think it is already 100% wide by default, so it's best to not have that line